connector-packager/connector_packager/jar_jdk_packager.py

```diff
@@ -46,8 +48,7 @@ def jdk_create_jar(source_dir, files, jar_filename, dest_dir):
     p = subprocess.Popen(args, cwd=abs_source_path)
     p.wait()
 
-    shutil.move(abs_source_path/Path(jar_filename), dest_dir/jar_filename)
+    shutil.move(abs_source_path/jar_filename, dest_dir/jar_filename)
```

Comment:
dest_dir is the absolute path right?

Reply:
Yeah, it's a `Path` object per the fn def